.github/ISSUE_TEMPLATE/marketing-request.md

```diff
@@ -0,0 +1,53 @@
+---
+name: Marketing Request
+about: Get help getting a message out to our community through coordinated outreach no email, social media, and other public channels.
+title: 'REQUEST: New communication about <topic>'
+labels: area/contributor-comms, sig/contributor-experience
+assignees: '@jonasrosland'
+---
+<!--
+ATTENTION: the Upstream Marketing Team have classifications on how urgent and important a message to the community is [outlined here][multichannel].
```

Comment:
```suggestion
ATTENTION: the Upstream Marketing Team have classifications on how urgent and important a message to the community is outlined here: https://git.k8s.io/community/communication/marketing-team/multichannel-communications.md#mapping-type-to-tier
```
It might be best to just link to the doc directly here, as its a comment it won't appear either when creating the issue off of it or in the issue after creation.